autosubmit/autosubmit.py

```diff
@@ -1965,7 +1965,7 @@ def prepare_run(expid, notransitive=False, start_time=None, start_after=None,
                 # Handling starting time
                 AutosubmitHelper.handle_start_time(start_time)
                 # Start after completion trigger block
-                AutosubmitHelper.handle_start_after(start_after, expid, BasicConfig())
+                AutosubmitHelper.handle_start_after(start_after, expid)
```

Comment:
Hmmm, the old code looks messy. We passed a new instance of `BasicConfig()` here. Then, in the function, the argument was named `BasicConfig`.

There is an import `BasicConfig` too that accesses a singleton. Now, which one was supposed to be used? An empty/new instance of `BasicConfig`? Or the global `BasicConfig`?

I'd need to check what's in `BasicConfig` before hitting the approve here, :bow: so won't be able to review it now (doing a 30 minutes round of reviews)

But thanks for spotting this and trying to fix! And with a test :muscle: